crates/ruff_python_ast/src/nodes.rs

```diff
@@ -845,6 +1059,221 @@ impl fmt::Debug for FStringElements {
     }
 }
 
+bitflags! {
+    #[derive(Default, Copy, Clone, PartialEq, Eq, Hash)]
+    struct TStringFlagsInner: u8 {
```

Comment:
Same here. Could we use a single `InterpolatedStringFlagsInner` instead of duplicating all this code